openfecwebapp/templates/partials/navigation.html

```diff
@@ -1,30 +1,33 @@
 <nav class="site-nav js-site-nav">
-  <ul id="site-menu" class="site-nav__list">
-    <li class="site-nav__item site-nav__item--with-dropdown" data-submenu="data">
-      <a href="{{ url_for('search') }}" class="site-nav__link is-parent">Campaign finance data</a>
-    </li>
-    <li class="site-nav__item">
-      <a href="{{ cms_url }}/registration-and-reporting/" class="site-nav__link">Registration and reporting</a>
-    </li>
-    {% if features.legal %}
-    <li class="site-nav__item">
-      <a href="{{ cms_url }}/legal-resources" class="site-nav__link">Legal resources</a>
-    </li>
-    {% else %}
-    <li class="site-nav__item is-disabled">
-      <a href="#" class="site-nav__link">Legal resources</a>
-    </li>
-    {% endif %}
-    <li class="site-nav__item utility-nav__item is-disabled">
-      <a href="#" class="site-nav__link">(About)</a>
-    </li>
-    <li class="site-nav__item utility-nav__item">
-      <a href="{{ cms_url }}/contact-us/" class="site-nav__link">Contact</a>
-    </li>
-    <li class="site-nav__item utility-nav__item">
-      <a href="{{ cms_url }}/calendar/" class="site-nav__link">Calendar</a>
-    </li>
-  </ul>
-  <button for="nav-toggle" class="js-nav-toggle site-nav__button site-nav__button--left" aria-controls="site-menu">Menu</button>
+  <div id="site-menu" class="site-nav__container">
+    <ul class="site-nav__panel site-nav__panel--main">
+      <li class="site-nav__item site-nav__item--with-dropdown" data-submenu="nav-data">
```

Comment:
Gaaaah. I totally misunderstood your comment. Yeah, that's a flub. Submitting a PR to fix.